doc/stdlib/dates.rst

```diff
@@ -136,6 +136,11 @@ alternatively, you could call ``using Dates`` to bring all exported functions in
    Returns a DateTime corresponding to the user's system
    time as UTC/GMT.
 
+.. function:: eps(::DateTime) -> Millisecond
+              eps(::Date) -> Day
+
+   Returns ``Millisecond(1)`` for ``DateTime`` values and ``Day(1)`` for ``Date`` values.
+
```

Comment:
This doc is kind of wrong. It is an extension to `Base.eps`, rather than `Dates.eps`. Not sure how our documentation system should handle this, but it seems like a general problem so it begs for a general solution.